bolt/include/bolt/Core/DebugData.h

```diff
@@ -798,6 +811,7 @@ class DwarfLineTable {
   // Returns DWARF Version for this line table.
   uint16_t getDwarfVersion() const { return DwarfVersion; }
 };
+
```

Comment:
stray line,I guess. There are others in this patch, but I'll write only once here